Feature/express eslint

Heyjp opened the pull request.

Heyjp commented:
Run `npm run lint` to get eslint to run.

sfiquet commented on package.json:
```diff
@@ -1,19 +1,55 @@
+    },
+    "eslintConfig": {
+        "env": {
+            "browser": true,
```
Shouldn't that be "node": "true" since this is for the server?

sfiquet commented on package.json:
```diff
@@ -1,19 +1,55 @@
+        "rules": {
+            "brace-style": [
+                "error",
+                "stroustrup"
```
Why?

sfiquet commented on package.json:
```diff
@@ -1,19 +1,55 @@
+            ],
+            "comma-dangle": [
+                "error",
+                "never"
```
I don't particularly like trailing commas, but Airbnb's style guide states that enabling dangling commas "leads to cleaner git diffs". I think they've got a point and it's worth trying it out.

sfiquet commented on package.json:
```diff
@@ -1,19 +1,55 @@
+                "off"
+            ],
+            "one-var": [
+                "off"
```
I like having one variable per line. It's more readable and there's the benefit on diffs.

From airbnb's style guide:
"Why? It’s easier to add new variable declarations this way, and you never have to worry about swapping out a ; for a , or introducing punctuation-only diffs. You can also step through each declaration with the debugger, instead of jumping through all of them at once."

sfiquet commented on package.json:
```diff
@@ -1,19 +1,55 @@
+                "warn"
+            ],
+            "no-var": [
+                "off"
```
I'd leave `no-var` on, `let` is a better choice now.

Heyjp commented:
Well these are all good points. However I only had them in to test eslint was working. I've now removed them and we are operating purely off the airbnb config. If there are changes / additions you would like to see we can have them written in. 

I've also removed `--fix` from the script, the idea is that we start coding to better standards and having our code auto updated we aren't going to learn as much as we could if we followed the command line. Let me know what you think?
